ocs_ci/utility/uninstall_openshift_logging.py

```diff
@@ -0,0 +1,75 @@
+"""
+Function to teardown the openshift-logging
+"""
+import logging
+
+from ocs_ci.ocs import constants, ocp
+
+logger = logging.getLogger(__name__)
+
+
+def uninstall_cluster_logging():
+    """
+    Function to uninstall cluster-logging from the cluster
+    1. Deletes the project "openshift-logging"
+        & "openshift-operators-redhat"
+    2. Deletes the CSC resource "cluster-logging-operator"
+        & "elasticsearch"
+    """
+
+    # Deleting the clusterlogging
+    clusterlogging_obj = ocp.OCP(
+        kind='ClusterLogging', namespace='openshift-logging'
+    )
+    assert clusterlogging_obj.delete(resource_name='instance')
+
+    # Validating all the pods are in Terminating state
+    pod_obj = ocp.OCP(
+        kind=constants.POD, namespace='openshift-logging'
+    )
+    ret = pod_obj.get()
+    assert ret, "Pods are in Running state."
+    logger.info("All pods are deleted")
+
+    # validating all PVCs got deleted
+    pv_obj = ocp.OCP(
+        kind=constants.PVC, namespace='openshift-logging'
+    )
+    pvc_check = pv_obj.get()
+    assert pvc_check, "PVCs are in Bound state"
```

Comment:
It never asserts since pvc_check will always contains data irrespective of PVC exists or not